### PR TITLE
404 page

### PR DESCRIPTION
**Pull request by Guitarninja1**
Screen Shot 2021-08-16 at 11 08 46 AM
Screen Shot 2021-08-16 at 11 08 25 AM

### CODE REVIEW

**Comment by kalm42** [comment]
You have conflicts to resolve. Your colors are off. The breadcrumbs are too wide. The spacing below 404 is too large.

File: pages/404.js [hunk 1]
```diff
@@ -1,7 +1,47 @@
...
-			<h1 className='404--oops'>OOPS!</h1>
+		<div className='fourzerofour--wrapper'>
+			<Header />
+			<div className='fourzerofour--title-wrapper'>
+				<h1 className='fourzerofour--title'>OOPS!</h1>
+			</div>
+			<ul className='fourzerofour--ul'>
+				<li>
+					<Link href='/'>
+						<a className='fourzerofour--li-home'>Home</a>
```
Comment (kalm42) [inline pages/404.js:15]: Close but there's a more optimal solution. `fourzerofour--li fourzerfour--li__with-hyphen` feel free to rename that but you get the idea.

File: pages/404.js [hunk 2]
```diff
@@ -1,7 +1,47 @@
...
+				</label>
+				<input
+					type='search'
+					id='search'
+					className='fourzerofour--search'
+					placeholder='Search'
+				/>
+				<button type='submit' className='fourzerofour--search-button'>
+					<Image src='/search.svg' alt='search' layout='fill' />
+				</button>
```
Comment (kalm42) [inline pages/404.js:43]: Submit what form?

File: styles/Global.css [hunk 3]
```diff
@@ -36,6 +36,10 @@ address {
 }
 p {
 	color: #989cac;
+	font-family: Mulish;
+	font-style: normal;
+	font-weight: normal;
+	line-height: 160%;
```
Comment (kalm42) [inline styles/Global.css:42]: Line height is commonly expressed in multiples. ie. `1.6` not `160%`

File: styles/Global.css [hunk 4]
```diff
@@ -260,7 +264,113 @@ a:hover {
...
+.fourzerofour--li-home {
+	font-size: 16px;
+	font-family: Mulish;
+	font-style: normal;
+	font-weight: normal;
+	line-height: 160%;
+	margin: 0 20px;
+}
+.fourzerofour--li-home::after {
+	content: ' 	\2000\2001\2001 ─';
```
Comment (kalm42) [inline styles/Global.css:335]: Why the spaces?

**Review by kalm42 — CHANGES_REQUESTED** [review]

**Comment by Guitarninja1** [comment]
Screen Shot 2021-08-16 at 12 50 34 PM
Screen Shot 2021-08-16 at 11 08 46 AM

**Comment by Guitarninja1** [comment]
fix breadcrumbs commit 
Mine
template

File: styles/Global.css [hunk 5]
```diff
@@ -260,23 +264,114 @@ a:hover {
...
+	margin: 0 20px;
+	display: flex;
+	justify-content: flex-end;
+	color: #989cac;
+}
+.breadcrumbs--li__with-hyphen::after {
+	content: '─';
+	opacity: 0.5;
+	position: absolute;
+	left: calc(100vw - 85px);
```
Comment (kalm42) [inline styles/Global.css:343]: Just putting this out there `right: 85px;`. Then there's a general rule that you always have at least two anchors. Top/left, bottom/right, ect. This isn't wrong. If you're having trouble positioning the hyphen remember to try setting the parent to `position: relative;`.

File: styles/Global.css [hunk 6]
```diff
@@ -260,23 +264,114 @@ a:hover {
...
+	background-color: transparent;
+	text-indent: 30px;
+	min-height: 60px;
+}
+.fourzerofour--search-icon {
+	position: absolute;
+	width: 20px;
+	height: 20px;
+	left: 19rem;
+	top: 34rem;
```
Comment (kalm42) [inline styles/Global.css:371]: Here we do want pixel values.

**Review by kalm42 — APPROVED** [review]

File: styles/Global.css (hunk 6 above)
Comment (Guitarninja1) [reply styles/Global.css:371]: haaaaaattte

**Review by kalm42 — APPROVED** [review]

**Pull request merged**